app/javascript/components/miq-about-modal.jsx

```diff
@@ -75,37 +76,93 @@ class MiqAboutModal extends React.Component {
       >
-        <AboutModal.Versions>
-          <AboutModal.VersionItem label={__('Version')} versionText={`${data.server_info.version}.${data.server_info.build}`} />
-          <AboutModal.VersionItem label={__('Server Name')} versionText={data.server_info.appliance || ''} />
-          <AboutModal.VersionItem label={__('Region')} versionText={data.region.region.toString()} />
-          <AboutModal.VersionItem label={__('Zone')} versionText={data.zone.name || ''} />
-          <AboutModal.VersionItem label={__('User Name')} versionText={data.identity.name} />
-          <AboutModal.VersionItem label={__('User Role')} versionText={data.identity.role} />
-          <AboutModal.VersionItem label={__('Browser')} versionText={browser.browser} />
-          <AboutModal.VersionItem label={__('Browser Version')} versionText={browser.version.toString()} />
-          <AboutModal.VersionItem label={__('Browser OS')} versionText={browser.OS} />
-          <br />
-          <a
-            style={{ color: 'white' }}
+        <ModalBody className="about-modal-body">
+          <p className="ModalItem">
+            <strong>
+              {`${__('Version')}`}
+            </strong>
+            &emsp;
+            {`${data.server_info.version}.${data.server_info.build}`}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('Server Name')} `}
+            </strong>
+            &emsp;
+            {data.server_info.appliance || ''}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('Region')} `}
+            </strong>
+            &emsp;
+            {data.region.region.toString()}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('Zone')} `}
+            </strong>
+            &emsp;
+            {data.zone.name || ''}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('User Name')} `}
+            </strong>
+            &emsp;
+            {data.identity.name}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('User Role')} `}
+            </strong>
+            &emsp;
+            {data.identity.role}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('Browser')} `}
+            </strong>
+            &emsp;
+            {browser.browser}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('Browser Version')} `}
+            </strong>
+            &emsp;
+            {browser.version.toString()}
+          </p>
+          <p className="ModalItem">
+            <strong>
+              {`${__('Browser OS')} `}
+            </strong>
+            &emsp;
+            {browser.OS}
+          </p>
+          <Button
+            kind="ghost"
+            className="plugins-button"
+            style={{ color: 'white', fontSize: '12px' }}
```

Comment:
Remove css in inline styles and add it in scss files. Color should be hash value. In this case #fff.